setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="mc_calculator",
+    version="0.1.0",
+    author="0xIkari",  # Replace with your name
+    author_email="ikari@nuclear-treestump.com",  # Replace with your email
+    description="A Minecraft recipe calculator",  # Brief description of your package
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",  # If your README is in Markdown
+    url="https://github.com/nuclear-treestump/mc-calculator",  # Replace with the URL of your repo
```

Comment:
May want to remove these boiler-plate comments to the right such as 'Replace with your name', etc.

Reply:
This is completed in latest PR